### PR TITLE
chore(Tailwind):setup[#187300090]

### PR DESCRIPTION
**Pull request by RWEMAREMY**
#### What does this PR do?

Setting up Tailwind

#### Description of Task to be completed?

-setting css framework to easy styling
 
#### How should this be manually tested?

- to run tailwind use "npm run build-css "

#### What are the relevant pivotal tracker/Trello stories?

- id of the project
  #187300090

### CODE REVIEW

File: dist/input.css
Comment (fab-ryan) [inline dist/input.css:1]: remove this dist folder

File: tailwind.config.ts [hunk 1]
```diff
@@ -13,8 +13,20 @@ const config: Config = {
...
+        sans: ["Inter", "sans-serif"],
+        serif: ["Georgia", "serif"],
+      },
+      darkMode: {
+        primary: "#1a202c",
+        secondary: "#2d3748",
+        accent: "#3b82f6",
+        background: "#0e172a",
+        text: "#ffffff",
+      },
```
Comment (fab-ryan) [inline tailwind.config.ts:39]: why did you add only dark mode only

**Review by fab-ryan — CHANGES_REQUESTED** [review]
the setup not complete

**Review by Mukunzijames — APPROVED** [review]

File: dist/input.css
Comment (RWEMAREMY) [reply dist/input.css:1]: my bad , with that said it's done!!!

**Review by fab-ryan — COMMENTED** [review]
Please if there is no dark mode then set up the colors right

File: tailwind.config.ts [hunk 2]
```diff
@@ -13,8 +13,26 @@ const config: Config = {
...
+      lightMode:{
+        primary:"#395B64",
+        background:"#FFFFFF",
+        text:"#000000"
+        
+      },
```
Comment (fab-ryan) [inline tailwind.config.ts]: no this is not what I mean I mean themes like if it;s blue, light blue, text colors

**Review by fab-ryan — CHANGES_REQUESTED** [review]
Correct that themes

**Review by Ruthuwamahoro — APPROVED** [review]

**Review by fab-ryan — APPROVED** [review]
Correct that themes

**Pull request merged**